src/routers/reactRouter.ts

```diff
@@ -30,14 +30,9 @@ const useReactRouter = (
   const getQuery = useCallback(
     (defaultQuery: Query) => {
       const query: Query = {}
-      for (const [key, value] of new URLSearchParams(search).entries()) {
-        const decodedValues = value
-          .split(QUERY_ARRAY_SEPARATOR)
-          .map((v) => decodeURIComponent(v))
```

Comment:
Is decoding not necessary anymore? Or is that handled by URLSearchParams already?

Reply:
Yes, URLSearchParams encodes (with `.toString()`) and decodes the query-string passed in the constructor.